src/components/hello-world/hello-world.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'c-hello-world',
+  styleUrl: 'hello-world.scss',
+  shadow: true
+})
+export class helloworld {
+
+  @Prop() PropHelloWorld: String = "hello-World";
```

Comment:
I prefer the property example is something that can be seen directly in the demo. This property will be translated in the element as `<c-hello-world prop-hello-world="A text" />`
Seems a bit cluttered for a simple example. What if change it to a more understandable property such as 'text', or 'name'.
So we can render something like: `<c-hello-world name="Jane Doe" /> `in the demo.
And according to stencil code guidelines, props are usually start with non-capital letters ;) So we don't mix up with classes (which started with capital letters)